### PR TITLE
changes after initial usability testing

### PR DESCRIPTION
**Pull request by jduss4**
reorders the report form pages, changes a little home page text, adds text to top of report page 1, adds report step links

ignore the name of the branch, I forgot to change it after I had started looking for the navigation background bug (which I ended up not addressing here)

### CODE REVIEW

File: pages/report.html [hunk 1]
```diff
@@ -6,31 +6,51 @@ <h1>Report an antitrust violation</h1>
...
-  <p>If you enter your email address or phone number, we will use it <em>only</em> if we need to respond to your report.</p>
+<section class="usa-section usa-section-report section-violation-type padding-bottom-05">
+  <h2>Thank you for taking the time to file a report.</h2>
+
+  <p>This form collections information that may help the Antitrust Division identify a potential antitrust violation. It has four sections:</p>
+
+  <ol>
+    <li>Type of antitrust violation: this helps us sort reports</li>
+    <li>Details: this information creates context and also helps us sort reports</li>
+    <li>Description: this lets you describe what happened and why you think the Antitrust Division should be aware of it</li>
```
[the remark this answers is not in the document]
Comment (melchoyce) [reply pages/report.html:17]: Like this?

```Description (required):```

[the remark this answers is not in the document]
Comment (jduss4) [reply pages/report.html]: That's what I just went for!  Let me know if you'd like it changed

**Review by melchoyce — APPROVED** [review]
Two things to update, but I think we can handle those later and push this now.

Updating these labels (@ameliaswong can you confirm this is correct?):
image

Bolding these: 
image

**Comment by ameliaswong** [comment]
@melchoyce All good. Thanks!

**Pull request merged**